Signal/src/contact/OWSContactsManager.m

```diff
@@ -264,7 +339,11 @@ - (void)buildSignalAccounts
 - (void)updateSignalAccounts:(NSArray<SignalAccount *> *)signalAccounts
 {
     AssertIsOnMainThread();
-    
+    if ([signalAccounts isEqual:self.signalAccounts]) {
```

Comment:
This is significant - we only notify of the change when the value of the accounts actually changes. Because a lot of things observe this notification, it's expensive to fire it all the time.

This is particularly helps with the "settling time" immediately after launch now that we set once from cache, and then set again a few seconds later when contact's fetch and then intersect, which usually won't result in a a change.